domain/schema/model/sql/0022-resource.sql

```diff
@@ -29,18 +29,13 @@ INSERT INTO resource_state VALUES
 
 CREATE TABLE resource (
     uuid TEXT NOT NULL PRIMARY KEY,
-    application_uuid TEXT NOT NULL,
+    resource_store_uuid TEXT,
     name TEXT NOT NULL,
```

Comment:
**todo:** I think we can remove the name too (it is in `resource_meta` and the entry point will be `application_resource` table.) 

I will be able to remove [this commit](https://github.com/juju/juju/commit/16323a4c0e7597c46293e9ed8713ef9f59b4643b) from my PR in state.